app/code/Magento/InventoryInStorePickupQuote/Model/GetWebsiteCodeByStoreId.php

```diff
@@ -0,0 +1,58 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\InventoryInStorePickupQuote\Model;
+
+use Magento\Framework\Exception\NoSuchEntityException;
+use Magento\Store\Api\StoreRepositoryInterface;
+use Magento\Store\Api\WebsiteRepositoryInterface;
+
+/**
+ * Provide website code by store id.
+ */
+class GetWebsiteCodeByStoreId
```

Comment:
I'm not sure if Store Pickup module is a good location for this kind of service.

Reply:
I agree. Does it real option just to mark it with '@internal'?